tests/scripts/thread-cert/message.py

```diff
@@ -294,8 +318,37 @@ def next_coap_message(self, code, uri_path=None, assert_enabled=True):
 
         return message
 
+    def last_mle_message(self, command_type, assert_enabled = True):
+        """Get the last Mle Message with specified type from existing capture.
+        Args:
```

Comment:
nit: leave a blank line before line #323 as the other places?

Reply:
fixed